db/migrate/20230221094708_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
```

Comment:
* Kindly create 5 different greetings as stated in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=create%205%20different%20greetings)